libs/libdaemon/include/daemon/Pipe.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
```

Comment:
Duplicate file.

Reply:
After short discussion decided to move `Pipe` to `libs/libcommon` (in this PR) and merge `dbms/src/Common` and `libs/libcommon` (as a separate commit)